packages/commonwealth/client/scripts/state.ts

```diff
@@ -216,6 +215,9 @@ const app: IApp = {
   isProduction: () =>
     document.location.origin.indexOf('commonwealth.im') !== -1,
   serverUrl: () => {
+    if (process.env.NODE_ENV === 'test') {
```

Comment:
This is extremely bad practice. We can't have conditions on test env in production code.

Reply:
Well this will be an issue, because this method uses the window object, which the mocha environment doesn't have. 

The other usage in this file is to stop the NewProfilesController from being created, because this causes a circular dependency.

Basically, NODE_ENV will always be run as 'production' for production, as well as it will be auto removed by webpack's Define plugin. Any alternative ideas on how we could resolve it this issue?

Reply:
Override the method directly (`app.serverUrl = ...`) before the test case.

Reply:
Good idea